install/helm/open-match/subcharts/open-match-telemetry/values.yaml

```diff
@@ -135,9 +135,12 @@ grafana:
   adminPassword: openmatch
   service:
     port: 3000
+  # Setup Grafana Alert Notification Channels
+  # https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L303
```

Comment:
This is an unstable reference to code.  (any change in the file before line 303 and it will break.)

What's this for, maybe link to documentation or no links?

Reply:
This is the instruction for Open Match users to setup Grafana alerts - e.g. slack notification, email alerts when certain metrics threshold is reached.